multicluster/controllers/multicluster/leader/clusterset_controller.go

```diff
@@ -199,14 +212,7 @@ func (r *LeaderClusterSetReconciler) updateStatus() {
 		status.Conditions = []mcv1alpha2.ClusterSetCondition{overallCondition}
```

Comment:
Just for my understanding, could you still answer my question if updateStatus() is called periodically and why?

Reply:
Hi @jianjuns updateStatus() will check the leader's connectivity status in the member cluster's ClusterSet controller, the status update action will be ignored if there is no change. The same function will check the member's connectivity through MemberClusterAnnounce in the leader cluster and the status will also not be updated if there is no change. 
I think the main purpose is reflect the connectivity status between leader and member clusters.